src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -837,10 +844,11 @@ def _job_settings(self, step_name: str, remote_wheels: list[str]) -> dict[str, A
             date_to_remove = self._get_test_purge_time()
             tags.update({"RemoveAfter": date_to_remove})
         return {
-            "name": self._name(step_name),
+            "name": self._name(workflow_name),
             "tags": tags,
             "job_clusters": self._job_clusters(job_clusters),
             "email_notifications": email_notifications,
+            "schedule": workflow.schedule,
```

Comment:
This works with `None`? Probably covered in directly by integration tests

Reply:
Yes: this dictionary is eventually used as `**kwargs` and schedule is an argument that defaults to `None`.